index.bs

```diff
@@ -524,13 +524,12 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         required boolean supported;
         required boolean smooth;
         required boolean powerEfficient;
-        
       };
     </pre>
 
     <pre class='idl'>
       dictionary MediaCapabilitiesDecodingInfo : MediaCapabilitiesInfo {
-        required MediaKeySystemAccess keySystemAccess;
+        MediaKeySystemAccess? keySystemAccess = null;
```

Comment:
Looking closer, I think I'm wrong. The member is nullable, but I don't think its optional? Or, if it is optional (due to lack of "required"), what does it mean to have an optional with a default value?

If possible, I like the idea that this only appears when making EME queries. Thoughts?

Reply:
Do we need to only have it appear when doing EME query? In practice, an EME query that doesn't return a `MediaKeySystemAccess` would actually says `supported = false`, right?

Reply:
True - I'm not saying we need its absence/presence signals anything we don't already know. But I thats also sort of my point - for non EME queries this will always be null - aesthetically dead weight. Is there an easy way to spec only having it for EME (i.e. only having it when it might be useful)?

Reply:
It's probably possible but I wouldn't recommend it. It doesn't provide any benefit and it would add complexity to the spec but most importantly to the implementation. Today, we can simply set the `keySystemAccess` to `null` unless we are getting a value but if we wanted to differentiate `null` and `undefined`, implementations will have to keep track of whether there was an EME request. Maybe it's not much to add but not really sure what that provides to the authors.

Reply:
Oh also, if we ever switch to an interface, we cannot make that difference: `undefined` can only be used here because it's a dictionary.

Reply:
Nullable SGTM